server/configuration.go

```diff
@@ -26,8 +30,24 @@ type alertConfig struct {
 	AlertManagerURL string
 }
 
-type configuration struct {
-	AlertConfigs map[string]alertConfig
+func (ac *alertConfig) IsValid() error {
+	if ac.Team == "" {
+		return errors.New("must set a Team")
```

Comment:
since you are adding new things, please move the `"github.com/pkg/errors"` to `errors`